test/libsolidity/util/BytesUtils.cpp

```diff
@@ -206,21 +206,28 @@ string BytesUtils::formatString(bytes const& _bytes, size_t _cutOff)
 	return os.str();
 }
 
-string BytesUtils::formatRawBytes(bytes const& _bytes)
+string BytesUtils::formatRawBytes(bytes const& _bytes, string _linePrefix, bool _withSignature)
```

Comment:
just a thought: Would it make sense to make the paramete ra `bytes _bytes` and modify it after the signature instead of creating `tail`?

Reply:
That's what I had at some point, but passing it by const-ref came out of another review. Also, I had some upcoming changes in mind (formatting weirdly-sized return values) which could make it mandatory to modularize this function.